requirements-extras.txt

```diff
@@ -1,4 +1,4 @@
-qiskit-ibm-provider>=0.6.1 # for submitting experiments to backends through the IBM provider
+qiskit-ibm-runtime>=0.15.0 # for submitting experiments to backends through the IBM provider
```

Comment:
Since nothing in the code or documentation actually imports `qiskit_ibm_runtime`, maybe we should leave it out of the extras and let the user install it separately? The motivation would be to avoid stray dependencies that cause conflicts like the one https://github.com/Qiskit-Extensions/qiskit-experiments/pull/1372 is working around with qiskit-ibm-provider. All the other packages here are imported in qiskit-experiments, just in parts that the user can choose not to use. On the other hand, one does often want to import qiskit-ibm-provider/runtime when using qiskit-experiments.

Reply:
Hmm, I see your point, but I thought the idea of having the `extras` dependencies was to install everything that a user could need for Experiments. Dynamics is also only used in tests, right? So in theory we can remove that from `extras` too. I'm not sure what's the best solution here.

Reply:
That's fair. Since the Qiskit install instructions now begin with installing `qiskit-ibm-runtime` and we also have a bunch of notes about it in the documentation, I've removed the dependency.